keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/DeliveryArtifact.kt

```diff
@@ -17,6 +18,14 @@ enum class ArtifactStatus {
   FINAL, CANDIDATE, SNAPSHOT, RELEASE, UNKNOWN
 }
 
+/**
+ * The ways in which artifacts can be sorted.
+ */
+enum class ArtifactSortByMethod(val type: String) {
```

Comment:
I couldn't come up with a better name for this enum... `ArtifactSortingMethod` didn't sound right either. Let me know if you have any suggestions.

Reply:
Suggestion: `ArtifactSortStrategy` or even just `SortStrategy`, since this is in the `artifacts` module already.

Reply:
I like `SortStrategy`

Reply:
shouldn't `branch` is enough? is there another way, other than `timestamp` to sort a branch's artifacts?

Reply:
My thinking was that, especially at this point, where this is very new, it was important to make the selection explicit. I've also had OSS in mind: they won't have support for sorting by branch and timestamp out of the box.